### ISSUE TITLE
separate time it takes training and validation to run on each epoch

### ISSUE THREAD

**Issue by ahundt** [fchollet/keras#4747]
Are there any suggested ways to separate the time it takes the training and validation time for each epoch?

looking through the [lstm_benchmark](https://github.com/fchollet/keras/blob/ff62eb251b04b8301e71aee970bdb157f2649fa9/examples/lstm_benchmark.py) shows how to get some information, as does the docs, but I can't seem to find something for this specific metric. Thanks!

**Comment by fchollet** [comment]
An easy way is to handle training and evaluation separately. E.g. call `fit` without any validation data, then call `evaluate` on your training data.
